python/ray/data/_internal/block_batching/iter_batches.py

```diff
@@ -97,8 +98,13 @@ def iter_batches(
             process. If set to greater than 0, a separate thread will be used to fetch
             the specified amount of formatted batches from blocks. This improves
             performance for non-CPU bound UDFs, allowing batch fetching compute and
-            formatting to be overlapped with the UDF. Defaults to 0 (no prefetching
-            enabled).
+            formatting to be overlapped with the UDF. Defaults to 1.
```

Comment:
updated the docs from 0 to 1, based on the current default value of `1` in the param definition.